app/views/output_elements/tables/_chart_142.html.haml

```diff
@@ -82,14 +87,14 @@
     %td{:data => {:gquery => "buildings_cooling_airconditioning_electricity_delta_in_investment_cost_table"}}
   %tr.odd
     %th= t "input_elements.buildings_cooling_collective_heatpump_water_water_ts_electricity_share"
-    %td.border-rgt{:data => {:gquery => "buildings_collective_heatpump_water_water_ts_electricity_present_in_investment_cost_table", :graph => :present}}
-    %td{:data => {:gquery => "buildings_collective_heatpump_water_water_ts_electricity_future_in_investment_cost_table"}}
-    %td{:data => {:gquery => "buildings_collective_heatpump_water_water_ts_electricity_delta_in_investment_cost_table"}}
+    %td.border-rgt{:data => {:gquery => "buildings_cooling_collective_heatpump_water_water_ts_electricity_present_in_investment_cost_table", :graph => :present}}
```

Comment:
I think here the queries of the individual technologies are used instead of the ones that correct for double counting. This seems to be happening with a lot of the combi technologies below as well.